README.md

```diff
@@ -1,44 +1,168 @@
+Modify this file to match your project and remove sections that don't apply.
 
+For example: Delete the testing section if the currect project doesn't require testing
+-->
 
-## Built With
+<div align="center">
 
-- Major languages
-- Frameworks
-- Technologies used
+  <img src="mv-readme-logo.png" alt="logo" width="240"  height="auto" />
+  <br/>
 
-## Live Demo (if available)
+  <h3><b>Microverse README Template</b></h3>
 
-[Live Demo Link](https://livedemo.com)
+</div>
 
+<!-- TABLE OF CONTENTS -->
 
-## Getting Started
+# 📗 Table of Contents
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
+<!-- PROJECT DESCRIPTION -->
 
-To get a local copy up and running follow these simple example steps.
+# 📖 [Project name] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[High level description]**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features
+
+> Describe between 1-3 key features of the application.
+
+- [Feature 1]
+- [Feature 2]
+- [Feature 3]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+[Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
+In order to run this project you need **[describe prerequisites]**:
+
+```sh
```

Comment:
Should we include it? It will be changing, so it is weird to have it in the template 🤔

Reply:
To me the goal of a template like this is show students **the standard way** of writing high level developer documentation for new projects. Depending on the project needs they can remove certain sections (which we have to specify, of course).

This **Prerequisites** section helps students think about the context, not just the project they worked on.

But I would love to know your thoughts! Also, what do you mean by "changing"?

Reply:
Valid points. I would argue that (1) students would need to change things anyway and (2) to know what's expected it helps to see an example. 

To find middle ground I've put those "examples" in a comment for now, which we can remove if it proves to be distracting!